raisinbread/README.md

```diff
@@ -29,6 +29,16 @@ that is not the case, replace all `mysql` commands below by the necessary values
 `mysql -u user -p -h host database_name`*
 
 ##### Sourcing SQL
+The script `tools/raisinbread_refresh.php` includes functionality that drops all existing tables in the
+database and sources all of the RaisinBread data automatically. To run this script, navigate to the LORIS
```

Comment:
I think ti would be useful to add a note here to the effect of "The script will preserve database settings such as API Keys and server configuration settings to simplify the process of switching between development environments.